param_templates/input_data_list.yaml

```diff
@@ -20,12 +20,8 @@ mom.input_data_list:
     ocean_topo_edit:
         $OCN_GRID == "tx0.25v1": "${DIN_LOC_ROOT}/ocn/mom/${OCN_GRID}/All_edits.nc"
     tempsalt:
-        $INIT_LAYERS_FROM_Z_FILE == "True" and $OCN_GRID == "gx1v6":
-            "${DIN_LOC_ROOT}/ocn/mom/${OCN_GRID}/WOA05_pottemp_salt.nc"
-        $INIT_LAYERS_FROM_Z_FILE == "True" and $OCN_GRID == "tx0.66v1":
-            "${DIN_LOC_ROOT}/ocn/mom/${OCN_GRID}/WOA05_pottemp_salt_180829.nc"
-        $INIT_LAYERS_FROM_Z_FILE == "True" and $OCN_GRID == "tx0.25v1":
-            "${DIN_LOC_ROOT}/ocn/mom/${OCN_GRID}/MOM6_IC_TS.nc"
+        $INIT_LAYERS_FROM_Z_FILE == "True" and $OCN_GRID in ["gx1v6", "tx0.66v1", "tx0.25v1"]:
+            "${INPUTDIR}/${TEMP_SALT_Z_INIT_FILE}"
```

Comment:
I think I understand from your email.  Thanks

Reply:
In fact, all other files paths should be changed from DIN_LOC_ROOT to INPUTDIR as well. I'll push another commit with that change.

Reply:
So INPUTDIR contains links to DIN_LOC_ROOT, but if a file isn't in DIN_LOC_ROOT then it is downloaded to INPUTDIR and not DIN_LOC_ROOT so the next time you need that file in another case it's going to download it again?

Reply:
By default, INPUTDIR is `${DIN_LOC_ROOT}/ocn/mom/${OCN_GRID}` . If the user changes INPUTDIR via user_nl_mom, then the MOM6 input files will be downloaded to the user-specified INPUTDIR. Ideally, when a user specifies a custom INPUTDIR, they should copy or symbollically link any MOM6 input file already available in `${DIN_LOC_ROOT}` to the user-specified INPUTDIR prior to submitting the case, so as to prevent the download of the files. Does this make sense?